guide/start/policies.md

```diff
@@ -9,169 +9,423 @@ children:
+above).
 
-This policy automatically scales the cluster up or down to be the right size for the cluster's current load. One server 
-is the minimum size allowed by the policy.
+The Tomcat server should be replaced by a new member of the cluster, and then the old failed member
+removed.
 
-Sitting idle, this cluster will only contain one server, but you can use a tool like [jmeter](http://jmeter.apache.org/) 
-pointed at the nginx endpoint to create load on the cluster. Download a jmeter test 
-plan [here](https://github.com/apache/incubator-brooklyn/blob/master/examples/simple-web-cluster/resources/jmeter-test-plan.jmx).
+You can view the list of Tomcat servers in the cluster with the command below (which drills into the  
+application named "Tomcat Cluster", then into its child entity named "Cluster", and then lists the 
+child entities):
 
-As load is added, Apache Brooklyn requests a new cloud machine, creates a new app server, and adds it to the cluster. 
-As load is removed, servers are removed from the cluster, and the infrastructure is handed back to the cloud.
+{% highlight bash %}
+br application "Tomcat Cluster" entity "Cluster" entity
+{% endhighlight %}
+
+<pre>
+ Id         Name            Type   
+ dYfUvLIw   quarantine      org.apache.brooklyn.entity.group.QuarantineGroup   
+ tOpMeYYr   Tomcat Server   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+ mgoRpkKH   Tomcat Server   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+</pre>
 
 
-### Under the Covers
+## Auto-scaling
 
-The `AutoScalerPolicy` here is configured to respond to the sensor
-reporting requests per second per node, invoking the default ```resize``` effector.
-By updating on the policy, you can configure it to respond to a much lower threshhold
-or set long stabilization delays (the period before it scales out or back).
+The `AutoScalerPolicy` attaches to a `Resizable` entity (in this case the `DynamicCluster`) and 
+dynamically adjusts its size in response to either keeping a metric within a given range, or in 
+response to Pool Cold and Pool Hot events. It adds/removes members (e.g. Tomcat instances) 
+automatically.
 
-At present the CLI does not support a command to update a policy.
+In this example policy, the `metricUpperBound` for requests per second per server is configured 
+very low at just 3. This makes it simple to trigger auto-scaling. The `resizeUpStabilizationDelay` 
+of 2 seconds means the load must be sustained for at least that length of time. The  
+`resizeDownStabilizationDelay` of 1 minute means there must be low load for a full one minute 
+before it will scale back down.
 
-However it is possible to manually suspend the policy, by 
+Find the URL of the load-balancer - see the `NginxController` sensor named `main.uri`.
+
+To generate load, you can use your web-browser by repeatedly refreshing that page. Alternatively,
+use a script like that below, or use a load generator like jmeter:
```

Comment:
Alternatively, use a script like that below -> Alternatively, use a script such as the one shown below (changing URL for the URL of your load-balancer)